aimmo-game/service.py

```diff
@@ -35,13 +35,13 @@ def to_cell_type(cell):
     return 0
 
 
-def player_dict(avatar):
+def player_dict(avatar, min_x, min_y):
     # TODO: implement better colour functionality: will eventually fall off end of numbers
     colour = "#%06x" % (avatar.player_id * 4999)
     return {
         'id': avatar.player_id,
-        'x': avatar.location.x,
-        'y': avatar.location.y,
+        'x': avatar.location.x-min_x,
```

Comment:
This is only used for the UI API (used by the JS front-end). The location for the API exposed to avatar's is based off each class' serialise method.